govulndb/govulndb.go

```diff
@@ -0,0 +1,124 @@
+package govulndb
+
+import (
+	"encoding/json"
+	"log"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/cheggaaa/pb/v3"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	vulndbURL = "https://storage.googleapis.com/go-vulndb"
```

Comment:
why not using https://osv-vulnerabilities.storage.googleapis.com/Go/all.zip ?
https://github.com/google/osv/blob/master/README.md#current-data-sources